packages/plugin-relay/src/utils/connections.ts

```diff
@@ -202,7 +202,7 @@ export function resolveArrayConnection<T>(
   };
 }
 
-function parseCurserArgs(options: ResolveOffsetConnectionOptions) {
+export function parseCurserArgs(options: ResolveOffsetConnectionOptions) {
```

Comment:
This should probably be renamed to parseCursorConnectionArgs before being exported